tst/EnergyPlus/unit/UnitaryHybridAirConditioner.unit.cc

```diff
@@ -515,18 +515,18 @@ TEST_F(EnergyPlusFixture, Test_UnitaryHybridAirConditioner_ValidateFieldsParsing
 {
     std::string idf_objects = delimited_string({
         "ZoneHVAC:HybridUnitaryHVAC,",
-        "MUNTERSEPX5000,          !- Name",
+        "Hybrid Unit 1,          !- Name",

```

Comment:
I'm not sure if this change is actually tested in the unit test, is it?  I'm OK with this simple IDD change anyway.  @mjwitte is there something I'm missing that this shouldn't go right in?

Reply:
Nope, nothing more to see here.

Reply:
It is related to the 'retaincase.' Previously, only object names in uppercase letters for ZoneHVAC:HybridUnitaryHVAC can be correctly recognized. With the change, the object name in mixed uppercase and lowercase letters can be correctly recognized using an existing unit test. It seems that additional lines are not showing here. Please refer to lines 579-603 for adding another object of ZoneHVAC:HybridUnitaryHVAC. In addition, a few checking function calls were added (lines 647-652) of this unit test.

Reply:
Yeah, it's fine.